src/url.js

```diff
@@ -0,0 +1,70 @@
+const FlowdockText = require('flowdock-text');
+
+function createTextNode(state, content, level) {
+  var token = new state.Token('text', '', 0);
```

Comment:
If you are using ES6, in ALM we're generally never using `var`, we always use `let`.